impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -437,6 +439,15 @@ std::shared_ptr<PipelineLibrary> ContextVK::GetPipelineLibrary() const {
   return pipeline_library_;
 }
 
+std::shared_ptr<CommandBufferQueue> ContextVK::GetCommandBufferQueue() const {
```

Comment:
Do we want a command buffer queue to be associated with a frame?  If we have concurrent frames being encoded couldn't this get jumbled up?

The way this is written couldn't one frame be written, then another one gets started written to and he's using the command buffer queue which gets flushed from the first frame?

Reply:
Yeah that part of this is definitely a bit fiddly. I don't think its currently possible for us to concurrent encode multiple frames, in that case as long as we reset the queue in submit it should be perfectly safe...